src/services/contractCategories/index.ts

```diff
@@ -0,0 +1,25 @@
+import { PrismaClient, ContractCategory } from '@prisma/client';
+import { CreateContractCategory, UpdateContractCategory } from './types';
+
+const prisma = new PrismaClient();
+
+export default class ContractCategoryService {
+  static async getAll(): Promise<ContractCategory[]> {
+    return prisma.contractCategory.findMany();
+  }
+
+  static async find(id: string): Promise<ContractCategory | null> {
```

Comment:
```suggestion
  static async find(query: FindQuery<ContractCategory>): Promise<ContractCategory | null> {
```

please see other services as example

Reply:
Done